2-scrape-metadata/index.js

```diff
@@ -27,15 +28,15 @@ var pageData = {
 };
 
 converter.fromFile( CSV_FILENAME, function( err, result ) {
-    var eisIdArray = result.map( function( obj ) {
-	return obj.eis_id;
+  var eisIdArray = result.map( function( obj ) {
+    return obj.eis_id;
   } );
-
-    eisIdArray.forEach( function( id ) {
-	request( 'https://cdxnodengn.epa.gov/cdx-enepa-II/public/action/eis/details?eisId=' + id, function( err, res, html ) {
+  Promise.map( eisIdArray, function( id ) {
```

Comment:
Won't this be firing many many requests in parallel? It might be better to chain the promises to fetching things one by one or in smaller batches.